dml-fused-activations.md rogue backslashes in struct definition

fdwr opened the pull request:
In https://docs.microsoft.com/en-us/windows/ai/directml/dml-fused-activations, there should be no rogue backslashes before/after the `_Maybenull_` SAL annotation:

Currently:
```
struct DML_CONVOLUTION_OPERATOR_DESC
{
    ...
    \_Maybenull\_ const DML_TENSOR_DESC* BiasTensor;
```

Expected:
```
struct DML_CONVOLUTION_OPERATOR_DESC
{
    ...
    _Maybenull_ const DML_TENSOR_DESC* BiasTensor;
```

opbld31 commented:
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/faq.md](https://github.com/MicrosoftDocs/windows-ai-docs/blob/master/docs/faq.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/windows/ai/faq?branch=pr-en-us-79) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C30%5C28ddff79-5366-c51c-8202-1b9e8beacb23%5CPullRequest%5C201811301908120429-79%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4N%2FMR%2FLZZuJZl2qsJU19wS7ZoRBiU%2BoqpbiXOlfVV6U%3D&st=2018-11-30T19%3A04%3A13Z&se=2018-12-31T19%3A09%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

fdwr committed:
dml-fused-activations.md rogue backslashes in struct definition

In https://docs.microsoft.com/en-us/windows/ai/directml/dml-fused-activations, there should be no rogue backslashes before/after the `_Maybenull_` SAL annotation:

```
struct DML_CONVOLUTION_OPERATOR_DESC
{
...
    \_Maybenull\_ const DML_TENSOR_DESC* BiasTensor;
```

fdwr commented:
@stevewhims Interesting. I just noticed this PR went into a separate repo, not the usual https://github.com/MicrosoftDocs/sdk-api, and no reviewers were auto-added.
